projects/plugins/jetpack/extensions/blocks/videopress/editor.scss

```diff
@@ -17,4 +17,8 @@
 			margin-right: 12px;
 		}
 	}
+}
+
+.videopress-block-hide {
+	display: none;
 }
```

Comment:
Just for consistency with the other files, I think we try to always include a new line at the end of a file.

Reply:
✅ I left it as it was previously but I felt it was a mistake 😅